packages/embed-services/src/EmbedUrl.ts

```diff
@@ -0,0 +1,134 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2023 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+
+export enum ContentType {
+  Look = 'LOOK',
+  Explore = 'EXPLORE',
+  Dashboard = 'DASHBOARD',
+}
+
+const THEMABLE_CONTENT: ContentType[] = [
```

Comment:
I presume this is from a standard we use somewhere on the frontend for constant names?

Reply:
hmm, it's how I am used to doing it. I believe I saw @bryans99 doing it similarly as well. I can look through a style guide to make sure though

Reply:
See https://google.github.io/styleguide/tsguide.html#identifiers

Reply:
I don't think Looker follows the Google style guide for TypeScript